src/i18n/index.ts

```diff
@@ -0,0 +1,52 @@
+import i18next from 'i18next';
+import {list} from '../plugin';
+import {EP_PHOVEA_CORE_LOCALE} from '../extensions';
+
+const DEFAULT_LANGUAGE = 'en';
+
+
+/**
+ *  Awaits the translation files registered at the EP_PHOVEA_CORE_LOCALE extension point
+ *  Initialize I18next with the translation files
+ */
+export async function initializeI18next() {
+  const plugins = await Promise.all(list(EP_PHOVEA_CORE_LOCALE).map((pluginDesc) => {
+    return pluginDesc.load().then((locale) => {
+      return {
+        lng: pluginDesc.lng,
+        ns: pluginDesc.ns,
+        resources: locale.factory(),
+        order: pluginDesc.order
+      };
+    });
+  }));
+
+  return i18next
+    .init({
+      debug: true,
```

Comment:
We should disable the debugging by default.

You could use the `__DEBUG__` boolean that is included in the _webpack.config.js_. An example can be found for the `__VERSION__` in the _index.ts_:

```ts
declare const __VERSION__: string;
export const version = __VERSION__;
```